prelude/prelude.purs

```diff
@@ -547,6 +660,7 @@ module Prelude
 
   infixr 5 <>
 
+  -- | Concatenation-like operation.
```

Comment:
How about "an associative operation"? If my understanding of what a semigroup is is correct, that's everything we can possibly say about `<>`.